creusot-contracts/src/logic/model.rs

```diff
@@ -21,3 +21,14 @@ impl<T: Model> Model for &mut T {
         (*self).model()
     }
 }
+
+impl<'a, T> Model for &'a [T] {
```

Comment:
I would like to define this directly on `[T]` but it introduces a bunch of issues related to `Sized`...